src/main/java/org/infoshareacademy/javaholics/route/RouteService.java

```diff
@@ -1,4 +1,149 @@
+        boolean error = false;
+        System.out.println("Podaj początek trasy : ");
+        route.setPlaceStart(scanner.nextLine());
+        System.out.println("Podaj koniec trasy : ");
+        route.setPlaceStop(scanner.nextLine());
+        System.out.println("Podaj długość trasy w km : ");
+        do{
+            error = false;
+            try {
+                route.setLength(scanner.nextDouble());
+                while (route.getLength() < 0 || route.getLength() == 0) {
+                    System.out.println("Podaj poprawną długośc trasy");
+                    route.setLength(scanner.nextDouble());
+                }
+            }catch (InputMismatchException e) {
+                System.out.println("Podaj poprawną długośc trasy");
+                error = true;
+                scanner.nextLine();}
+        }while (error);
+        System.out.println("Początek twojej trasy jest : " + route.getPlaceStart() + " , a koniec jest " + route.getPlaceStop());
+        System.out.println("Całkowita długość trasy to " + route.getLength() + " km");
+    }
+
+    public void routeLevelOfDifficulty() {
+        boolean error = false;
+        System.out.println("Podaj poziom trudności trasy :");
+        System.out.println("1 - Łatwy");
+        System.out.println("2 - Średni");
+        System.out.println("3 - Wymagający");
+        System.out.println("4 - Bardzo trudny");
+        int levelOfDifficulty;
+        do{
+            error = false;
+            try {
+                levelOfDifficulty = scanner.nextInt();
+                while (levelOfDifficulty != 1 && levelOfDifficulty != 2 && levelOfDifficulty != 3 && levelOfDifficulty != 4) {
+                    System.out.println("Wybierz poprawny poziom trudności");
+                    levelOfDifficulty = scanner.nextInt();
+                }
+                if (levelOfDifficulty == 1) {
+                    System.out.println("Poziom trasy - Łatwy");
+                    route.setDifficulty(RouteDifficulty.EASY);
+                }
+                if (levelOfDifficulty == 2) {
+                    System.out.println("Poziom trasy - Średni");
+                    route.setDifficulty(RouteDifficulty.MEDIUM);
+                }
+                if (levelOfDifficulty == 3) {
+                    System.out.println("Poziom trasy - Wymagający");
+                    route.setDifficulty(RouteDifficulty.EXTREME);
+                }
+                if (levelOfDifficulty == 4) {
+                    System.out.println("Poziom trasy - Bardzo trudny");
+                    route.setDifficulty(RouteDifficulty.HARD);
+                }
+            } catch (InputMismatchException e) {
+                System.out.println("Wybierz poprawny poziom trudności trasy");
+                error = true;
+                scanner.nextLine();
+            }
+        } while (error);
+        System.out.println("Proponowany poziom trudności stworzonej trasy to " + route.getDifficulty());
```

Comment:
jesli odwołasz się do enuma, jak proponiwałem to wtedy
System.out.println("Poziom trudności trasy to : " + route.getDifficulty().getDifficultyTranslation());